processing/src/main/java/org/apache/carbondata/processing/store/CarbonFactDataHandlerColumnar.java

```diff
@@ -644,10 +556,9 @@ private void resetBlockletProcessingCount() {
   }
 
   /**
-   * This class will hold the holder objects and manage producer and consumer for reading
-   * and writing the blocklet data
+   * This class will hold the encoded table pages.
    */
-  private final class BlockletDataHolder {
+  private final class TablePageList {
```

Comment:
This class only holds some table pages, size of which equals to number of cores, but not blocklet. So rename it.